src/main/java/org/team1540/robot2018/commands/climber/WinchOut.java

```diff
@@ -0,0 +1,27 @@
+package org.team1540.robot2018.commands.climber;
+
+import edu.wpi.first.wpilibj.command.Command;
+import org.team1540.robot2018.Robot;
+import org.team1540.robot2018.Tuning;
+
+public class WinchOut extends Command {
+
+  public WinchOut() {
```

Comment:
are any of these classes (WinchOut, WinchIn, TapeOut, TapeIn, etc, i know there's more) actually getting used? i think it's better style to actually use them, but if we aren't doing that, delete them all.

Reply:
`WinchOut` is used [here](https://github.com/flamingchickens1540/pandora-titan-2018/pull/8/files#diff-ee56816ca2f61bdffc0991b1df2b8b8fR43). All the others are already deleted.

Reply:
Of course, `WinchOut` could easily be replaced with a `SimpleCommand` since that appears to be what we're doing anyway.